### DISCUSSION TITLE
Criando Apps Web com Angular e UiKit

### DISCUSSION THREAD

**Discussion by lxxzh** [balta-io/forum#211] [Tira Dúvidas] [answered]
Bom dia @andrebaltieri ,  após a Aula 33 do curso acima, Edição do Perfil Parte 2, usando via terminal o **NG SERVE -O, o navegador não exibe o contéudo, a página fica em branco.** Já vi e revi as aulas e não sei dizer oque errei, pois o código está igual aos arquivos do curso. Poderia me dar um apoio?

<img width="960" alt="image" src="https://user-images.githubusercontent.com/106619427/211036021-2dce9db8-2a5d-4722-83c7-df8b3bbbff2f.png">

**Comment by lxxzh** [comment, marked as answer]
@andrebaltieri pode encerrar esta dúvida, verifiquei as rotas e eu estava colocando o canActivate: [AuthService] na rota errada. Problema Solucionado.

**Reply by andrebaltieri** [reply]
Perfeito!!!
